src/Dfe.PlanTech.Web/Controllers/PagesController.cs

```diff
@@ -31,9 +31,13 @@ public async Task<IActionResult> Index([FromServices] GetPageQuery query, Cancel
 
     [Authorize]
     [HttpGet("/{route?}")]
-    public async Task<IActionResult> GetByRoute(string route, [FromServices] GetPageQuery query, CancellationToken cancellationToken, string param = "")
+    public async Task<IActionResult> GetByRoute(string route, [FromServices] GetPageQuery query, CancellationToken cancellationToken)
     {
         string slug = GetSlug(route);
+        string param = "";
+
+        if (TempData[slug] is string) param = TempData[slug] as string ?? "";
```

Comment:
C# is fantastic, so this line you could do as 

```
        if (TempData[slug] is string s) param = s;
```

instead, which IMO looks a bit cleaner.

Reply:
Good to know you can do that!

Implemented in: [950beaa](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/216/commits/950beaacc13e623113783a5ccdef6dd31fb265b2)